src/clj_jargon/users.clj

```diff
@@ -9,13 +9,21 @@
                                       UserAO]
            [org.irods.jargon.core.pub.domain UserGroup]))
 
+(def ^:private user-type-mapping
+  {"rodsuser"   :user
+   "groupadmin" :group-admin
+   "rodsadmin"  :admin
+   "rodsgroup"  :group
+   "unknown"    :unknown
+   })
+
 (defn user
   [{^UserAO user-ao :userAO} username]
   (otel/with-span [s ["user" {:attributes {"irods.user" username}}]]
     (try
       (let [jargon-user (.findByName user-ao username)]
         {:id (.getId jargon-user)
-         :type :user ;; better if we use the UserTypeEnum for rodsuser/groupadmin/rodsadmin/rodsgroup/unknown
+         :type (get user-type-mapping (.getTextValue (.getUserType jargon-user)) :unknown)
```

Comment:
This change is to enable validating that a user is a group admin or full admin. Nothing in data-info uses the type field that I can find, and nothing I could find anywhere checked any value but `:none` (to indicate the user doesn't exist), so I think this shouldn't cause any issues.